core/eth/client.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/shopspring/decimal"
 )
 
+const (
+	// PrepaidAggregatorName is the string name of said contract.
```

Comment:
Quick attempt to improve "PrepaidAggregatorName is Chainlink's Ethereum contract for aggregating (what's being aggregated?) "

Not a blocker